resources/mediaplayer/src/main/java/org/mobicents/media/server/impl/resource/mediaplayer/audio/wav/WavTrackImpl.java

```diff
@@ -266,19 +242,18 @@ private int readPacket(byte[] packet, int offset, int psize) throws IOException
         offset += 46;
```

Comment:
Where does this magic number 46 come from? Can we make it a constant and explain why it assumes this particular value?